src/ui/devtools/components/body.tsx

```diff
@@ -30,11 +30,23 @@ function Body({data, actions}: BodyProps) {
             reset: () => actions.resetDevDynamicThemeFixes(),
         });
 
-    function onTextRender(node) {
+    function onTextRender(node: HTMLTextAreaElement) {
         textNode = node;
         if (!state.errorText) {
             textNode.value = wrapper.fixesText;
         }
+        node.addEventListener('keydown', (e) => {
+            if (e.key === 'Tab') {
+                e.preventDefault();
+                const start = node.selectionStart;
+                const end = node.selectionEnd;
+                const before = node.value.substring(0, start);
+                const after = node.value.substring(end);
+                node.focus();
+                node.value = `${before}\t${after}`;
```

Comment:
Let's replace `\t` with 4 spaces.

Reply:
For some reason it would just add 1 space... Render body is resetting it to 1 space.......

Reply:
![image](https://user-images.githubusercontent.com/25481501/81706273-35517400-9470-11ea-9727-e5c9aacfff3e.png)
Bad idea to use 4 spaces